FHEM/14_SD_WS.pm

```diff
@@ -992,9 +990,9 @@ sub SD_WS_WH2SHIFT($){
   <b>Generierte Readings:</b><br><br>
   <ul>(verschieden, je nach Typ des Sensors)</ul>
   <ul>
-  	<li>battery: (low oder ok)</li>
-    <li>channel: (Der Sensor Kanal)</li>
-    <li>humidity: (Luftfeuchte (1-100)</li>
+  	<li>batteryState (low oder ok)</li>
+    <li>channel (Der Sensor Kanal)</li>
+    <li>humidity (Luftfeuchte (1-100)</li>
 		<li>state (T: H:)</li>
```

Comment:
Ich glaube hier müsste `<li>state (T: H: W:)</li>`  wie in der englischen Version hin.